src/browser/monkey-menu.js

```diff
@@ -153,10 +159,26 @@ function onHashChange(event) {
 }
 
 
+// Import a new user script
+// TODO: Add onImportUserScript to monkey menu when file picker bugs are fixed
```

Comment:
I find it counterproductive and confusing to have this whole thing written half in one style and half in another.  I'm also not keen on the awkward wording in the UI.

Write it so that it works well, now, complete.  If and only if in the (distant, if ever) future something different becomes possible, then and only then write code to do that.